sdk/aqueduct/table_artifact.py

```diff
@@ -130,6 +130,23 @@ def get(self, parameters: Optional[Dict[str, Any]] = None) -> pd.DataFrame:
         else:
             raise AqueductError("Artifact does not have table.")
 
+    def head(self, n: int = 5, parameters: Optional[Dict[str, Any]] = None) -> pd.DataFrame:
+        """Previews TableArtifact as a actual dataframe.
```

Comment:
English nit: could you change this to "Returns a preview of the table artifact"?